src/smart_markers/mod_smart_markers_backend.erl

```diff
@@ -48,6 +59,33 @@ get_chat_markers(HostType, To, Thread, TS) ->
     Args = [HostType, To, Thread, TS],
     mongoose_backend:call_tracked(HostType, ?MAIN_MODULE, ?FUNCTION_NAME, Args).
 
+%% @doc remove all entries for a given domain
+-spec remove_domain(mongooseim:host_type(), jid:lserver()) -> term().
+remove_domain(HostType, Domain) ->
+    Args = [HostType, Domain],
+    mongoose_backend:call(HostType, ?MAIN_MODULE, ?FUNCTION_NAME, Args).
+
+%% @doc remove all stored interactions with a given user
+-spec remove_user(mongooseim:host_type(), jid:jid()) -> term().
+remove_user(HostType, User) ->
+    Args = [HostType, User],
+    mongoose_backend:call(HostType, ?MAIN_MODULE, ?FUNCTION_NAME, Args).
+
+%% @doc remove all markers a user was ever sent
```

Comment:
```suggestion
%% @doc remove all markers a user has ever sent
```

Reply:
No no, in this case we're removing based on the `to_jid`, this one is to remove for example all markers sent to a room in particular, but it could be used not only for rooms.

Reply:
Ooh, I see. Then "has ever been sent"? 😄

Reply:
Fixed 😄

Reply:
thx